pmoves/tests/conftest.py

```
@@ -156,6 +161,56 @@ async def close(self):  # pragma: no cover - trivial
+    try:
+        nats_real = importlib.import_module("nats")
+    except ImportError:
+        pytest.skip("nats-py not installed")
+    nc = await nats_real.connect(nats_url)
+    try:
+        yield nc
+    finally:
+        try:
+            await nc.close()
+        except Exception:
+            pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)